packages/kbn-test/src/functional_test_runner/run_check_ftr_code_owners.ts

```diff
@@ -36,15 +32,22 @@ export async function runCheckFtrCodeOwnersCli() {
       const missingOwners = new Set<string>();
 
       // cache codeowners for quicker lookup
-      const reversedCodeowners = getPathsWithOwnersReversed();
+      log.info('Reading CODEOWNERS file');
+      const codeOwnersEntries = getCodeOwnersEntries();
 
       const testFiles = await getRepoFiles(TEST_DIRECTORIES);
+      log.info(`Checking ownership for ${testFiles.length} test files (this will take a while)`);
+
       for (const { repoRel } of testFiles) {
-        const owners: CodeOwnership = getCodeOwnersForFile(repoRel, reversedCodeowners);
-        if (owners === undefined || owners.teams === '') missingOwners.add(repoRel);
+        const owners = getOwningTeamsForPath(repoRel, codeOwnersEntries);
```

Comment:
I think we need the empty string check, see [here](https://github.com/elastic/kibana/pull/202704/files#r1867798621)

Reply:
No longer needed. Empty owners are [stripped earlier](https://github.com/elastic/kibana/blob/ea4fcd8bb1fb4e22040afec0d235542811ba4e81/packages/kbn-code-owners/src/code_owners.ts#L70) in the process.